pkgs/by-name/aq/aquamarine/package.nix

```diff
@@ -0,0 +1,79 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  fetchFromGitHub,
+  hwdata,
+  hyprutils,
+  hyprwayland-scanner,
+  libdisplay-info,
+  libdrm,
+  libffi,
+  libGL,
+  libinput,
+  libseat,
+  mesa,
+  nix-update-script,
+  pixman,
+  pkg-config,
+  udev,
+  wayland,
+  wayland-protocols,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "aquamarine";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "hyprwm";
+    repo = "aquamarine";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-rux5XA+ixI0fuiQGSOerLKxsW2D8cfjmP1B7FY24xF8=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    hyprwayland-scanner
+    pkg-config
+  ];
+
+  buildInputs = [
+    hyprutils
+    libdisplay-info
+    libdrm
+    libffi
+    libGL
+    libinput
+    libseat
+    mesa
+    pixman
+    udev
+    wayland
+    wayland-protocols
+  ];
+
+  depsBuildBuild = [ hwdata ];
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  cmakeBuildType = "RelWithDebInfo";
+
+  dontStrip = true;
```

Comment:
I haven't checked aquamarine's package size with stripping disabled, but if left unchecked it could grow a lot. Hyprland without stripping is nearly 500 MB.

Perhaps we should leave it undefined, and users can override it in.

Reply:
```bash
du /nix/store/HASH-aquamarine-0.1.1 -h 
```

shows that it is around 18 MB (without stripping)
with stripping it is 1 MB.

Reply:
Let's remove the line then.